doc/content/example/index.md

```diff
@@ -0,0 +1,3 @@
+# Example Problems
+
+Example problems will be added to the BlackBear repository in the future. In the meantime, checkout the BlackBear assessment problems at [here](assessment/assessment_index.md).
```

Comment:
This all looks good, except that 'checkout' isn't a verb (but 'check out' is)...

I'll let this slide, though.